types/color/color-tests.ts

```diff
@@ -1,4 +1,4 @@
-import Color = require('color');
+import Color from 'color';
```

Comment:
Please put this in a separate PR and cite where `color` changed its module format to justify this